app/assets/javascripts/ajax-video-search.js

```diff
@@ -0,0 +1,6 @@
+$(function() {
+  $('#video-search input').keyup(function() {
+    $.get($('#video-search').attr('action'), $('#video-search').serialize(), null, 'script');
+    return false;
+  });
+});
```

Comment:
I don't think you actually rebased onto master. You may have rebased onto your local master, but that was probably a couple commits out of sync. Make sure to switch to master, then `git pull upstream master` before you rebase, else you'll be rebasing onto nothing new.